tests/framework_tornado/_target_application.py

```diff
@@ -250,8 +220,6 @@ def make_app(custom=False):
         (r"/init", InitializeHandler),
         (r"/html-insertion", HTMLInsertionHandler),
         (r"/bad-get-status", BadGetStatusHandler),
-        (r"/force-cat-response/(\S+)/(\S+)/(\S+)", ProcessCatHeadersHandler),
-        (r"/304-cat-response/(\S+)/(\S+)", ProcessCatHeadersHandler, {"response_code": 304}),
```

Comment:
Not sure but this is probably still needed for testing 304 responses if the previous comment about 304 is valid?